icons/nixos/nixos-original-wordmark.svg

```diff
@@ -0,0 +1,2 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#7EBAE4" d="M18.096 57.082 8.12 74.362l-2.33-3.95 2.69-4.627-5.34-.014L2 63.798l1.162-2.019 7.602.024 2.731-4.71 4.601-.011zm.766 13.813h19.953l-2.255 3.993-5.352-.015 2.658 4.632-1.14 1.972-2.329.002-3.78-6.594-5.444-.012-2.311-3.978zm11.613-7.572L20.5 46.043 25.084 46l2.663 4.643 2.683-4.618h2.277l1.167 2.016-3.822 6.571 2.713 4.72-2.29 3.991z" clip-rule="evenodd" fill-rule="evenodd"></path><path fill="#5277C3" d="m14.496 64.2 9.975 17.28-4.584.043-2.663-4.642-2.683 4.618-2.277-.001-1.167-2.016 3.821-6.57-2.712-4.722 2.29-3.99zm11.587-7.615-19.954-.001 2.255-3.992 5.353.015-2.658-4.632 1.14-1.972L14.546 46l3.78 6.595 5.445.011 2.31 3.979zm.778 13.853 9.977-17.28 2.33 3.95-2.69 4.627 5.341.014 1.138 1.973-1.162 2.018-7.601-.024-2.732 4.71-4.601.012z" clip-rule="evenodd" fill-rule="evenodd"></path><path d="M68.355 53.027H65.84v11.418c0 1.612.065 4.16.162 6.999h-.097c-1.258-2.355-2.677-4.612-3.677-6.16l-7.902-12.257h-3.452v21.868h2.516V63.477c0-1.613-.064-4.645-.161-7.45h.097c1.87 3.257 2.967 5.063 4 6.611l7.902 12.257h3.128V53.027zm4.79 21.868h2.71V59.284h-2.71v15.611zM74.5 56.994c.935 0 1.677-.742 1.677-1.677 0-.936-.742-1.678-1.677-1.678-.936 0-1.677.742-1.677 1.678 0 .935.741 1.677 1.677 1.677zm18.234 2.29h-3.096l-3.806 5.741h-.097l-3.677-5.741h-3.355l5.58 7.612v.097l-5.773 7.902h3.096l4.064-6.193h.097l4.064 6.193h3.355l-5.903-8.031v-.097l5.451-7.483zm10.48-5.868c-5.134 0-9.355 4.134-9.355 11.014 0 6.879 4.221 11.013 9.355 11.013s9.355-4.134 9.355-11.013c0-6.88-4.221-11.014-9.355-11.014zm0 2.146c3.822 0 6.731 3.156 6.731 8.868 0 5.711-2.909 8.867-6.731 8.867s-6.731-3.156-6.731-8.867c0-5.712 2.91-8.868 6.731-8.868zm10.417 3.443c0 3.03 1.578 4.67 5.239 6.154 2.809 1.136 3.945 2.145 3.945 4.228 0 2.525-2.052 3.787-4.766 3.787-1.42 0-3.029-.22-4.796-.82l-.284 2.177c1.546.568 3.25.789 4.954.789 4.229 0 7.763-2.146 7.763-6.47 0-2.997-1.83-4.449-5.522-5.932-2.462-.978-3.661-2.02-3.661-4.197 0-2.114 1.83-3.282 4.134-3.282 1.357 0 2.84.347 3.882.758l.284-2.146c-1.199-.505-2.683-.758-4.324-.758-3.881 0-6.848 2.24-6.848 5.712z"></path></svg>
+
```

Comment:
Keep only 1 extra line at the end (same in the other files).
```suggestion
```

Reply:
oh sorry, I saw wrong because of my editor